cdap-ui/app/cdap/components/AbstractWidget/DateRangeWidget/index.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useState } from 'react';
+import withStyles, { WithStyles } from '@material-ui/core/styles/withStyles';
+import { IWidgetProps } from 'components/AbstractWidget';
+import ExpandableTimeRange from 'components/TimeRangePicker/ExpandableTimeRange';
+import { DatePickerStyles } from 'components/AbstractWidget/DateTimeWidget';
+
+const styles = () => {
+  return {
+    ...DatePickerStyles(),
```

Comment:
If we are just using `DatePickerStyles` then we can directly use in the interface extends right?
Like,
```interface IDateRangeWidgetProps extends IWidgetProps<null>, WithStyles<typeof DatePickerStyles> { ... }```